doc/.vuepress/config.js

```diff
@@ -1,55 +1,63 @@
 module.exports = {
-    title: 'Gerbil Scheme',
-    description: '',
-    themeConfig: {
-        nav: [
-          { text: 'Home', link: '/' },
-          { text: 'Guide', link: '/guide/' },
-          { text: 'Tutorials', link: '/tutorials/' },
-          { text: 'Reference', link: '/reference/' },
-        ],
-        sidebarDepth: 2,
-        sidebar: {
-          '/guide/': [
-            {
-              collapsable: false,
-              title: 'Guide',
-              children: ['', 'intro', 'getting-started', 'emacs', 'shell', 'env-vars', 'package-manager', 'ffi', 'build', 'profiler', 'bootstrap', 'nix', 'r7rs', 'srfi']
-            }
-          ],
-          '/tutorials/': [
-             {
-               collapsable: false,
-               title: 'Tutorials',
-               children: ['', 'languages', 'kvstore', 'proxy', 'httpd']
-             }
-          ],
-          '/reference/': [
```

Comment:
It seems none of these lines really changed except for indentation. Can you fix it so the commit is clearer?

Thanks :)

Reply:
Indentation was not consistent, so I fixed while adding the svg loader config. Are you asking to clarify in the commit message, or remove the changes to indentation?

Reply:
it's ok, we can keep the new indentation; looks better.

Reply:
I see. Consistent is better :)